bn-apps/cordapp-updates-distribution/corda-updates-shell/src/main/kotlin/net/corda/businessnetworks/cordaupdates/shell/ConsoleRepositoryListener.kt

```diff
@@ -0,0 +1,67 @@
+package net.corda.businessnetworks.cordaupdates.shell
+
+import org.eclipse.aether.AbstractRepositoryListener
+import org.eclipse.aether.RepositoryEvent
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+/**
+ * Simple maven resolver repository listener that logs everything into the console
+ * The implementation is kindly borrowed from https://github.com/eclipse/aether-demo/blob/master/aether-demo-snippets/src/main/java/org/eclipse/aether/examples/util/ConsoleRepositoryListener.java
+ */
+class ConsoleRepositoryListener : AbstractRepositoryListener() {
```

Comment:
Maybe call this LoggingRepositoryListener - it's not guaranteed to go to the console.

Reply:
Renamed